magpylib/_lib/getBvector.py

```diff
@@ -63,13 +64,18 @@ def getBv_magnet(type,MAG,DIM,POSo,POSm,ANG=[],AX=[],ANCH=[]):
     ANCH=[] : length M list of Nx3 numpy arrays float [mm]
         Anchor positions of M subsequent rotations applied ot the N-sized POSm and
         the implicit source orientation.
+    
+    Nphi0=50 : integer gives number of iterations used when calculating diametral
+        magnetized cylindrical magnets.
     """
 
     N = len(MAG)
 
     # set field type
     if type == 'box':
         Bfield = Bfield_BoxV
+    elif type == 'cylinder':
+        Bfield = Bfield_CylinderV
     elif type == 'sphere':
         Bfield = Bfield_SphereV
     else:
```

Comment:
Raise a Runtime Warning instead of printing inside of functions. Printing may get drowned out, and users can catch and handle the situation better based on their interpreter settings when detecting a warning.

```python
from warnings import warn
 warn('Bad Type or WIP', RuntimeWarning)
```